eZ/Publish/API/Repository/Tests/LocationServiceAuthorizationTest.php

```diff
@@ -427,9 +427,11 @@ public function testDeleteLocationWithSubtreeThrowsUnauthorizedException()
         $roleService = $repository->getRoleService();
 
         $role = $roleService->loadRoleByIdentifier('Editor');
+        $roleDraft = $roleService->createRoleDraft($role);
 
         $removePolicy = null;
-        foreach ($role->getPolicies() as $policy) {
+        /** @var \eZ\Publish\API\Repository\Values\User\PolicyDraft $policy */
```

Comment:
```
namespace eZ\Publish\API\Repository\Values\User;

/**
 * This class represents a draft of a role.
 *
 * @property-read \eZ\Publish\API\Repository\Values\User\PolicyDraft[] $policies
 */
abstract class RoleDraft extends Role
{
}
``` 
If we can add property-read then this comment will not be needed. Also, in https://github.com/ezsystems/ezplatform-rest/pull/21